notify/receivers.go

```diff
@@ -246,6 +247,23 @@ func (am *GrafanaAlertmanager) TestReceivers(ctx context.Context, c TestReceiver
 	return newTestReceiversResult(testAlert, append(invalid, results...), now), nil
 }
 
+func (am *GrafanaAlertmanager) buildSingleIntegration(r *GrafanaIntegrationConfig, tmpl *templates.Template) (*Integration, error) {
```

Comment:
This is [moved](https://github.com/grafana/grafana/blob/106eceab55627cb7d41acab2517f39a0d344e1bb/pkg/services/ngalert/notifier/alertmanager.go#L359-L374) from `grafana/grafana` which seems correct since it's used only for receiver testing.